### ISSUE TITLE
Stuck in "Waiting to scan"

### ISSUE THREAD

**Issue by Gammelgreggan** [jotta/jotta-cli-issues#93]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
0.6.18443

**Description of problem:**
With the new release the "jotta-cli status" command only returns "Waiting for scan to start" for my backup folders, even if I initiates "jotta-cli scan" command.
The folders to backup is not backed up.

The scan interval is set to 12 hours and the folders to scan is reachable.

**Expected:**
Folders should be backed up and "jotta-cli status" should eventually return "Fodlers up to date".

**jotta-cli status (`jotta-cli status`):**
See above.

```

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
```

```

**Traceback**
```

```

**Additional info:**

**Comment by Kimbsen** [comment]
Hi, could you contact support@jottacloud.com and we can look at the details of your case there?
Please include the logfile and output of jotta-cli status in your email.

**Comment by ostgardh** [comment]
Hi. 
I have same problem.
Have found solution?

**Comment by Gammelgreggan** [comment]
Hi!
I do not see this anymore, do not remember what solved it.
Could be that jotta-cli did not have permissions to the folder.
I changed user in config file for jotta-cli, and chmod the folders. The folder was a SMB folder.

Or the problem was that the SMB was on a NAS and I was not able to give the jotta-cli user permissions so I copied all files to another folder where it had permissions.

I hope any of this will help you.
